module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -0,0 +1,133 @@
+# Advanced scRNA-seq Cheatsheet
+
+#### The tables below consist of valuable functions or commands that will help you through this module.
+##### Each table represents a different library/tool and its corresponding commands.
+
+##### You may also be interested in the following additional cheatsheets:[Introduction to R and Tidyverse cheatsheet](https://github.com/AlexsLemonade/training-modules/blob/master/module-cheatsheets/intro-to-R-tidyverse-cheatsheet.pdf) and the [Introduction to Single-Cell RNA sequencing cheatsheet](https://github.com/AlexsLemonade/training-modules/blob/master/module-cheatsheets/scRNA-seq-cheatsheet.pdf).
```

Comment:
I accepted this change, but might revert. I'm remembering now the reason I chose _not_ to do the `raw/` links is because it will forcibly download the cheatsheet, rather than linking to an in-browser view (I wonder if this is browser-specific, but I get same behavior in safari+firefox).

I generally like to avoid surprise downloads, so I might update text to say plainly that these are download links, or include a link to download vs view.